bessctl/conf/testing/run_module_tests.bess

```diff
@@ -42,6 +42,10 @@ from pybess import protobuf_to_dict as pb_conv
 # Env variable used to run a single test
 test_name = $TEST_NAME!'*'
 
+default_path = os.path.join(os.path.dirname(os.path.realpath(sys.argv[0])),
+                            'conf/testing/module_tests')
```

Comment:
Shouldn't this just be `'module_tests`?  I forget precisely how this worked before all the changes but the current code (below) just adds `module_tests`...

Reply:
Ah, the tests moved. I just copied this from the original impl. Will update

Reply:
Fixed